cmd/release-controller/sync.go

```diff
@@ -547,6 +668,50 @@ func (c *Controller) syncAccepted(release *Release) error {
 			}
 		}
 	}
+	// If bugzilla bugs were fixed by a PR approved by their QA contact, move bug to verified
```

Comment:
This does not belong here.  You may not add synchronous code that is not reentrant to this location.

I probably need to see quick write up of the logic you are intending to add, how the loop should work, and what the boundaries are.  You can't put things in the main sync loop generally that can't be deferred or delegated.  This *probably* (without understanding exactly what your logic is) is a secondary publish loop that should be completely independent, like the auditor.

You'll need to describe your logic through before I can recommend that.

Reply:
In general this code path has to finish in O(ms) and the majority of these calls don't satisfy that.

Reply:
This code is essentially like a publish step, and it's supposed to improve workflow for QE.

Basically, a developer can request a review from the QA contact on the bugzilla bug associated with the PR using `/bugzilla cc-qa`. If the QA contact gives a `/lgtm` (or gives an approving review on a repo where approval review == lgtm), when a nightly build that includes the changes gets accepted, the bug is automatically moved to the VERIFIED state without needing extra steps from QE.